modules/porous_flow/tests/density/GravDensity01.i

```diff
@@ -0,0 +1,149 @@
+# Trivial test of PorousFlowTotalGravitationalDensityFullySaturatedFromPorosity
+# Porosity = 0.1
+# Solid density = 2500
+# Fluid density = 1000
+# Expected bulk density = 2500 * (1 - 0.1) + 1000 * 0.1 = 2350
+
+[Mesh]
+  type = GeneratedMesh
+  dim = 3
+  xmin = 0
+  xmax = 1
+  ymin = 0
+  ymax = 1
+  zmin = -1
+  zmax = 0
+  nx = 1
+  ny = 1
+  nz = 1
+[]
+
+[GlobalParams]
+  block = 0
+  PorousFlowDictator = dictator
+[]
+
+[Modules]
+  [./FluidProperties]
+    [./simple_fluid]
+      type = SimpleFluidProperties
+      thermal_expansion = 0
+      bulk_modulus = 2E9
+      density0 = 1000
+    [../]
+  [../]
+[]
+
+[Variables]
+  [./pp]
+    [./InitialCondition]
+      type = ConstantIC
+      value = 0
+    [../]
+  [../]
+[]
+
+[Functions]
+[]
+
+[Kernels]
+  [./dummy]
+    type = Diffusion
+    variable = pp
+  [../]
+[]
+
+[BCs]
+  [./p]
+    type = PresetBC
+    variable = pp
+    boundary = 'front back'
+    value = 0
+  [../]
+[]
+
+[AuxVariables]
+  [./density]
```

Comment:
In which case, having a lower bulk density will be a good idea.  And maybe a more complicated porosity function - i'll let you decide as this could get really complicated quite quickly!